### ISSUE TITLE
QC mni_normalized_anatomical may be incorrect 

### ISSUE THREAD

**Issue by nimzodisaster** [FCP-INDI/C-PAC#997]
CPAC v 1.4.1
This is labeled MNI_normalized_anatomical, but it appears to be an epi image. Moreover, is shows a bad registration, but when I load up the warped structural or warped mean functional, both appear registered to MNI correctly.

![mni_anat_a](https://user-images.githubusercontent.com/19696085/57952053-ded34900-78a0-11e9-8832-8a7bf0cc738b.png)

Also received this error during processing:
```
	 [Node] Running "_montage_a0" ("CPAC.utils.function.Function")

[!] QC Interface: Had a problem with creating the axial montage for mni_anat_a.png

Details:index 182 is out of bounds for axis 2 with size 182.This error might occur because of a registration error encountered while using ANTs.                   Please refer to the image located in your working directory for more insight
```

**Comment by sgiavasis** [comment]
Thanks @nimzodisaster - we are currently addressing this. Will post an update about when the fix is released.

**Comment by sgiavasis** [comment]
Just need to confirm this is no longer happening.

**Comment by sgiavasis** [comment]
Still happening. Addressing now.

**Comment by sgiavasis** [comment]
Fix will be in 1.6.1!

**Comment by sgiavasis** [comment]
This is in v1.6.1.
